portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,9 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I did not learn how to ride a bike until 13.', 'My favorite color is red.', 'Puerto Rico is the oldest colony in the world at the moment, and I am one of its residents.', 
+    'I have a mild obsession with instant ramen with boiled eggs.', 'My favorite saga is Star Wars.', 'I used to practice Taekwondo until I had kneee surgery.', 
+    'I have 4 siblings; 2 brothers and 2 sisters.', 'I am a professional procrastinator!'];
```

Comment:
Try to keep max line length at 100 chars for JS files. Add new lines where necessary to meet this requirement.